script.keymap/addon.xml

```diff
@@ -1,26 +1,34 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="script.keymap" name="Keymap Editor" version="1.1.3+matrix.1" provider-name="takoi,pkscout">
+<addon id="script.keymap" name="Keymap Editor" version="1.1.4" provider-name="takoi,pkscout">
 	<requires>
 		<import addon="xbmc.python" version="3.0.0"/>
-		<import addon="script.module.kodi-six" version="0.1.3.1" />
 		<import addon="script.module.defusedxml" version="0.6.0+matrix.1" />
 	</requires>
 	<extension point="xbmc.python.script" library="default.py"/>
 	<extension point="xbmc.addon.metadata">
-		<news>
-v1.1.3
-- Added support for v19 Matrix
-- Fix so script only backups user defined keymaps if trying to save new keymap file
-- Moved all text into language file for easier internationalization
+		<news>v1.1.4
+- fixed depreciacted translatepath call
+- removed dependancy on kodi-six
+- updated language files
 		</news>
 		<assets>
         	<icon>icon.png</icon>
         </assets>
-		<summary lang="en_GB">An editor for keymap files</summary>
-		<description lang="en_GB">Keymap Editor is a GUI for configuring mappings for remotes, keyboard and other inputs supported by Kodi. Note: existing user defined key mappings will be renamed to .bak.</description>
 		<platform>all</platform>
 		<license>GPL-3.0-only</license>
 		<forum>https://forum.kodi.tv/showthread.php?tid=168767</forum>
 		<source>https://github.com/tamland/xbmc-keymap-editor</source>
+		<summary lang="de_DE">Ein Editor für Keymap-Dateien</summary>
```

Comment:
Can you change the source also to your repo? I had to go to the repo-scripts and trace the history of the matrix branch to find out where the matrix version was